.github/ISSUE_TEMPLATE/Question.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Question
+about: Ask a quesion of how-tos.
+title: Question
+label: question
+---
+
+Before asking a question, use this link to see the questions that were already asked: https://github.com/Microsoft/vscode-java-pack/issues?q=is%3Aissue+label%3Aquestion+
```

Comment:
How about searching all the issue for similar topics instead of limiting to `label:question` ?